iOS/Flows/People/PeopleViewController.swift

```diff
@@ -167,8 +169,47 @@ class PeopleViewController: DiffableCollectionViewController<PeopleCollectionVie
 
     // MARK: Data Loading
 
-    override func getAllSections() -> [PeopleCollectionViewDataSource.SectionType] {
-        return PeopleCollectionViewDataSource.SectionType.allCases
+    override func retrieveDataForSnapshot() async -> [PeopleSection : [PersonItem]] {
+        var data: [PeopleSection: [PersonItem]] = [:]
+
+        let connections = (try? await GetAllConnections().makeRequest(andUpdate: [], viewsToIgnore: [])) ?? []
+
+        var connectedPeople: [Person] = []
+        await connections.asyncForEach { connection in
+            guard let userId = connection.nonMeUser?.personId else { return }
+
+            if let updatedPerson = await PeopleStore.shared.getPerson(withPersonId: userId) {
+                let person = Person(person: updatedPerson, connection: connection)
+                connectedPeople.append(person)
+            }
+        }
+        self.allPeople.append(contentsOf: connectedPeople)
+
+        let users = PeopleStore.shared.users
+        let unconnectedUsers = users.filter { user in
+            guard !user.isCurrentUser else { return false }
+
+            // Filter out users who are already connected
+            let isConnected = connectedPeople.contains(where: { connectedPerson in
+                return user.personId == connectedPerson.personId
+            })
+            return !isConnected
+        }
+        let unconnectedPeople = unconnectedUsers.map { unconnectedUser in
+            return Person(person: unconnectedUser, connection: nil)
+        }
+        self.allPeople.append(contentsOf: unconnectedPeople)
```

Comment:
Getting all of the existing jibber users that are related to you, but not yet connected.